python/lammps/mliap/__init__.py

```diff
@@ -25,8 +26,10 @@
     raise OSError("Unable to locate python shared library") from e
 
 if not pylib.Py_IsInitialized():
-    raise RuntimeError("This interpreter is not compatible with python-based mliap for LAMMPS.")
+    warnings.warn("This interpreter is not compatible with python-based MLIAP for LAMMPS. "
```

Comment:
Why this change to a warning instead of an error?

Reply:
The idea here was that users could run LAMMPS with a pickled model even without having mliappy properly installed. The exception was getting in the way of this so it was changed to a warning and if mliappy isn't installed then `activate_mliappy`, `load_model`, and `load_unified` are not imported into the `lammps.mliap` module (they could still potentially be imported from the `loader` submodule but using them would result in undefined behavior).

Reply:
Not sure if that is a good idea. I personally prefer a clean error and forcing people to do a proper installation.
But this is "your" code that you will have to support and thus the choice is yours.

Reply:
We can speak with @lubbersnick about it as it was his rationale. I agree that the implementation here could be cleaner; it should probably raise an error if someone does try to invoke one of the `mliappy` functions without the right installation.

Reply:
Hey, we can make it error if someone tries to active the MLIAP module. But there is no need to make it error to prevent people from accessing -other- things in lammps.mliap. Like @Boogie3D said the other objects can be used to export models for lammps even when the 'reverse' python driving lammps mode doesn't work (which is on most macs right now). From what I know fixing this can require installing python from source to adjust the build options, which is a few steps beyond what we should expect from a typical user.

Reply:
Honestly, I don't care much, I mostly was curious. Also, I am not going to make any more comments on the Python/Mac/Conda/etc situation in public.

What worries me much more is that it is not possible for me to run most of the new example inputs currently and thus verify my modifications to avoid memory corruption.